WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -159,3 +159,17 @@
     <%= f.submit 'Save', class: "btn btn-primary" %>
   <% end %>
 </div>
+
+<script>
+$('.date_picker.form-control').on('dp.error', function(e){
```

Comment:
This is the only place where they input a birthdate and we save, so I guess it's fine.

Unless you mean everywhere as in every input in every page. In that case, that's a LOT of work which is probably not worth at this point.

I was using the .has-error, but the calendar also turned red, which didn't make much sense.